{
  "verb": "get",
  "path": "/api/2.0/accounts/{account_id}/network-connectivity-configs/{network_connectivity_config_id}",
  "description": "Gets a network connectivity configuration.",
  "parameters": {
    "account_id": {
      "in": "path",
      "required": true,
      "type": "uuid",
      "description": "."
    },
    "network_connectivity_config_id": {
      "in": "path",
      "required": true,
      "type": "uuid",
      "description": "Your Network Connectvity Configuration ID."
    }
  },
  "responses": {
    "200": {
      "network_connectivity_config_id": {
        "description": "Databricks network connectivity configuration ID."
      },
      "account_id": {
        "description": "The Databricks account ID that hosts the credential."
      },
      "name": {
        "description": "Example"
      },
      "\"my-example-ncc\"": {
        "description": "The name must match the regular expression"
      },
      "^[0-9a-zA-Z-_]{3,30}$": {
        "description": "."
      },
      "region": {
        "type": "string",
        "description": "The region for the network connectivity configuration. Only workspaces in the same region can be attached to the network connectivity configuration."
      },
      "egress_config": {
        "description": "The network connectivity rules that apply to network traffic from your serverless compute resources."
      },
      "default_rules": {
        "description": "You can find the stable network information of your serverless compute resources here."
      },
      "creation_time": {
        "description": "Example"
      },
      "1691436121706": {
        "description": "Time in epoch milliseconds when this object was created."
      },
      "updated_time": {
        "description": "Example"
      },
      "1691436121777": {
        "description": "Time in epoch milliseconds when this object was updated."
      }
    },
    "401": {
      "error_code": "UNAUTHORIZED",
      "description": "The request does not have valid authentication credentials for the operation."
    },
    "404": {
      "error_code": "NOT_FOUND",
      "description": "Operation was performed on a resource that does not exist."
    },
    "500": {
      "error_code": "INTERNAL_SERVER_ERROR",
      "description": "Internal error."
    }
  }
}